### DISCUSSION TITLE
Hot key to open

### DISCUSSION THREAD

**Discussion by megmoo** [tprouvot/Salesforce-Inspector-reloaded#416] [Q&A]
I see at the bottom of the window ...

[ctrl+alt+i] to open

... but when I use that combination on Chrome, it doesn't work. Is there perhaps some setting I need to change to make this work? Does anyone have any suggestions? Thanks!

**Comment by tprouvot** [comment]
Hi @megmoo 
Could you try on a new chrome profile with only inspector reloaded installed ?

**Comment by csr040** [comment]
For me it works with this combination [alt + i]. @tprouvot, since there is a hot key to open shouldn't there also be one to close it? Currently I don't see anything for that.

**Comment by tprouvot** [comment]
@csr040 Currently there is not shortcut to close the popup but I might use the same one and close it if already opened.
Anyway this topic about shortcut should be closed by the end of the month with [this new feature](https://tprouvot.github.io/Salesforce-Inspector-reloaded/how-to/?h=sho#customize-extensions-shortcuts).
Already available in the beta version

**Comment by tprouvot** [comment]
Hi @megmoo, @csr040 
Hotkey to open the extension will be available in the 1.26 version.
I should release the beta mid Jan 25
